src/modules/0.0.18/aws_cloudfront/entity/distribution.ts

```diff
@@ -0,0 +1,90 @@
+import {
```

Comment:
nit: I think we use 2 spaces instead of 4 everywhere else. We don't enforce in the lint and it is not that important, but can we keep the convention? VSCode let's you change the configuration for it